powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -435,7 +455,13 @@ public boolean sendCampaign(Long campaignId) throws PushServerClientException {
         }
     }
 
-    private <T> ObjectResponse<T> checkStatus(TypeReference typeReference, HttpResponse response) throws IOException, PushServerClientException {
+    /**
+     * Checks if format of http response is valid
+     *
+     * @param typeReference reference on type of response body from which map into JSON
+     * @param response prepared http response
+     */
+    private <T> ObjectResponse<T> checkHttpResponseFormat(TypeReference typeReference, HttpResponse response) throws IOException, PushServerClientException {
```

Comment:
Why did you rename this method and added incorrect comment about what it does? This method explicitly checks HTTP status code value (it checks nothing else) and serializes response object accordingly. Name `checkStatus` was OK, maybe `checkHttpStatus` would be better, but these renames are unnecessary...